src/components/Footer/index.css

```diff
@@ -0,0 +1,47 @@
+footer {
+  margin-top: 50px;
+}
+
+.footer-links {
+  display: flex;
+  font-family: cubanoregular;
+}
+
+@media (max-width: 620px) {
+  .footer-links {
+    flex-wrap: wrap;
+  }
+}
+
+.footer-links a {
+  padding: 40px;
+  flex-basis: 100%;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+  margin: 0;
+}
+
+.footer-links a span {
+  display: inline-block;
+  font-size: 18px;
+  transition: all 150ms var(--timing-function);
+  transform: scale(1);
+}
+
+.footer-links a:hover span {
+  transform: scale(1.2);
+}
+
+.footer-links a.nature-of-code {
+  background: var(--ct-red);
```

Comment:
I wonder if we should be using the verbose versions:
```css
.footer-links a.nature-of-code {
  background: #F35956;
  background: var(--ct-red, #F35956);
}
```

Sadly, it kind of ruins the nature of centralising the variables...